internal/actions/setvar_test.go

```diff
@@ -46,3 +53,115 @@ func TestSetvarInit(t *testing.T) {
 		}
 	})
 }
+
+var invalidSyntaxAtoiError = "invalid syntax"
+var warningKeyNotFoundInCollection = "key not found in collection"
+
+func TestSetvarEvaluateErrors(t *testing.T) {
```

Comment:
IIUC, there is a fix related to `-` should there be a new test for it?

Reply:
Yes, `Numerical operation - with existing negative variable` and `Numerical operation - with existing variable` are the two test cases added to ensure that the fix related to `-` is working as expected.

Reply:
Ah I think I got confused by the test function name, it looks like not all of these are errors

Reply:
Yep, sorry about that. The test evolved over time and is not anymore just about errors. Renamed it to `TestSetvarEvaluate`